File: specification/vmware/resource-manager/Microsoft.AVS/stable/2021-12-01/examples-java/PrivateClouds_Get.java
import com.azure.core.util.Context;

/** Samples for PrivateClouds GetByResourceGroup. */
public final class Main {
    /*
     * x-ms-original-file: specification/vmware/resource-manager/Microsoft.AVS/stable/2021-12-01/examples/PrivateClouds_Get.json
     */
    /**
     * Sample code: PrivateClouds_Get.
     *
     * @param manager Entry point to AvsManager.
     */
    public static void privateCloudsGet(com.azure.resourcemanager.avs.AvsManager manager) {
        manager.privateClouds().getByResourceGroupWithResponse("group1", "cloud1", Context.NONE);
    }
}

File: specification/vmware/resource-manager/Microsoft.AVS/stable/2022-05-01/examples-js/PrivateClouds_Get.js
const { AzureVMwareSolutionAPI } = require("@azure/arm-avs");
const { DefaultAzureCredential } = require("@azure/identity");

/**
 * This sample demonstrates how to Get a private cloud
 *
 * @summary Get a private cloud
 * x-ms-original-file: specification/vmware/resource-manager/Microsoft.AVS/stable/2022-05-01/examples/PrivateClouds_Get.json
 */
async function privateCloudsGet() {
  const subscriptionId = "00000000-0000-0000-0000-000000000000";
  const resourceGroupName = "group1";
  const privateCloudName = "cloud1";
  const credential = new DefaultAzureCredential();
  const client = new AzureVMwareSolutionAPI(credential, subscriptionId);
  const result = await client.privateClouds.get(resourceGroupName, privateCloudName);
  console.log(result);
}

privateCloudsGet().catch(console.error);
